### PR TITLE
Several improvements to help run on cluster

### PR DESCRIPTION
**Pull request by mikewalch**
CC Paths file is now copied to HDFS.  Improved configuration.
Reduced replication factor on HDFS files and partitioned data.
Files are also checked to see if they exist before they are
copied.

### CODE REVIEW

**Comment by keith-turner** [comment]
Can you give me a high level description of the changes related to reversing domains?

**Comment by mikewalch** [comment]
Domains are no longer reversed because it was harder to create Accumulo table splits when row keys start with com, org, etc than if they were not reversed.  If they are not reversed, there is a new issue of many hosts starting with www but I think this could be improved by making the URI of `http://www.apache.org/page` look like `apache.org:www/page`

File: modules/data/src/main/java/io/fluo/webindex/data/spark/IndexEnv.java [hunk 1]
```diff
@@ -71,21 +76,59 @@ public IndexEnv(DataConfig dataConfig, SparkConf sparkConf) throws IOException {
...
 
-  public void initAccumuloIndexTable() throws AccumuloSecurityException, AccumuloException,
-      TableNotFoundException, TableExistsException {
-    if (conn.tableOperations().exists(dataConfig.accumuloIndexTable)) {
-      conn.tableOperations().delete(dataConfig.accumuloIndexTable);
+  public static SortedSet<Text> getDefaultSplits() {
+    SortedSet<Text> splits = new TreeSet<>();
+    String splitsFn = "src/main/resources/default-splits.txt";
+    try {
+      try (BufferedReader br = new BufferedReader(new FileReader(new File(splitsFn)))) {
```
Comment (keith-turner) [inline modules/data/src/main/java/io/fluo/webindex/data/spark/IndexEnv.java]: Could use classloader resource methods to read this.   Should work no matter what the CWD is.

Comment (mikewalch) [inline modules/data/src/main/java/io/fluo/webindex/data/spark/IndexEnv.java]: Fixed in 3bd97ee412b.  Also fixed a unit test that had the same issue.

**Comment by keith-turner** [comment]
+1

**Pull request merged**